pulpcore/app/serializers/content.py

```diff
@@ -183,7 +185,7 @@ class ContentChecksumSerializer(serializers.Serializer):
 
     class Meta:
         model = models.Content
-        fields = base.ModelSerializer.Meta.fields + (
+        fields = tuple(i for i in base.ModelSerializer.Meta.fields if i != "pulp_last_updated") + (
```

Comment:
```suggestion
        exclude = ("pulp_last_updated",)
        fields = base.ModelSerializer.Meta.fields + (
```
No need for fancy tuple conditions.

Reply:
I tried with the exclude (not sure if I did it wrong) but got the following exception:
```
Traceback (most recent call last):
...
  File "/usr/local/lib/python3.8/site-packages/rest_framework/serializers.py", line 1117, in get_field_names
    assert not (fields and exclude), (
AssertionError: Cannot set both 'fields' and 'exclude' options on serializer ArtifactSerializer.
```

Reply:
We can exploit the set's functionality for this, right?

https://github.com/pulp/pulpcore/blob/a33612b89f48985448f5315dc5c67aa54f94d196/pulpcore/app/serializers/publication.py#L324

Reply:
in container plugin we have even a richer example
https://github.com/pulp/pulp_container/blob/main/pulp_container/app/serializers.py#L214
https://github.com/pulp/pulp_container/blob/main/pulp_container/app/serializers.py#L378